### PR TITLE
renamed FHSDKPhone81 to FHSDKPortable

### PR DESCRIPTION
**Pull request by edewit**
Makes more sense as it's usable for windows phone / windows 8.1 and windows 10

### CODE REVIEW

**Comment by edewit** [comment]
@wei-lee can you review? Renamed because the word phone doesn't make sense on desktop or xbox

**Comment by wei-lee** [comment]
reviewed

**Comment by edewit** [comment]
@wei-lee thanks

File: copyLibs.bat [hunk 1]
```diff
@@ -11,10 +11,13 @@ COPY /B FHSDKPhone\bin\Release\FHSDK.xml Dist\lib\wp81\
...
-COPY /B FHSDKPhone81\bin\Release\FHSDK.xml Dist\lib\wpa81\
-COPY /B FHSDKPhone81\bin\Release\FHSDKPhone81.dll Dist\lib\wpa81\
-COPY /B FHSDKPhone81\bin\Release\FHSDKPhone81.XML Dist\lib\wpa81\
+COPY /B FHSDKPortable\bin\Release\FHSDK.dll Dist\lib\wpa81\
+COPY /B FHSDKPortable\bin\Release\FHSDK.xml Dist\lib\wpa81\
+COPY /B FHSDKPortable\bin\Release\FHSDKPortable.dll Dist\lib\wpa81\
+COPY /B FHSDKPortable\bin\Release\FHSDKPortable.XML Dist\lib\wpa81\
+
+COPY /B FHSDKPortable\bin\Release\FHSDKPortable.dll "Dist\lib\portable-win81+wpa81\"
+COPY /B FHSDKPortable\bin\Release\FHSDKPortable.XML "Dist\lib\portable-win81+wpa81\"
```
Comment (vibe13) [inline copyLibs.bat]: @edewit @wei-lee  Hi guys, what is the difference between folders "Dist\lib\wpa81\" (which now would contain FHSDK and FHSDKPortable) and "Dist\lib\portable-win81+wpa81\" (which now would contain FHSDKPortable only) ? Thanks!

**Pull request merged**